### DISCUSSION TITLE
Test drive a Pyboard offline?

### DISCUSSION THREAD

**Discussion by opticaldrive** [micropython/micropython#12765] [General]
It seems like the Test drive a Pyboard feature on the MicroPython website has been offline for some time.
Is it just my computer or is it offline?
Thanks!

**Comment by peterhinch** [comment]
That hasn't worked for a long time. I guess it should either be fixed or humanely euthanised. @dpgeorge ?

**Reply by opticaldrive** [reply]
Interesting.
I think I played with it just last month......
Thanks!

**Comment by Josverl** [comment]
Or replaced by embedding or linking to a few equivalent esp32 and rp2 wokwi projects.

https://wokwi.com/micropython

Adding additional starter projects should be simple

**Reply by picpic020960** [reply]
+1
but no stm32 with micropython

**Reply by opticaldrive** [reply]
I guess that would work too.

**Reply by Josverl** [reply]
> but no stm32 with micropython 

No not yet, but IMO the strength of micropython is the portability across ports, so that is not blocking

**Comment by andrewleech** [comment]
The other officially supported demo is https://micropython.org/unicorn/
